app/assets/scripts/components/documents/single-edit/step-closeout/index.js

```diff
@@ -97,13 +97,27 @@ StepCloseout.propTypes = {
 
 const MAX_ABSTRACT_WORDS = 250;
 
+function wordCount(value) {
+  const keys = Object.keys(value);
+
+  if (keys.includes('children')) {
+    return value.children.reduce((sum, child) => sum + wordCount(child), 0);
+  }
+
+  if (keys.includes('text')) {
+    const trimmed = value.text.trim();
+    return trimmed ? trimmed.split(/\s+/).length : 0;
+  }
+
+  return 0;
+}
```

Comment:
This could be made a bit more flexible by also accepting a value that is a `string`. Another option would be to rename the function to make it clear that it expects a rich field value.

Reply:
Yes, agreed. I renamed the function as I didn't want to introduce additional complexity that we might not need